pygmt/helpers/decorators.py

```diff
@@ -416,7 +416,7 @@ def fmt_docstring(module_func):
     <BLANKLINE>
     Parameters
     ----------
-    data : str or numpy.ndarray or pandas.DataFrame or xarray.Dataset or geo...
+    data : str, numpy.ndarray, pandas.DataFrame, xarray.Dataset, or geo...
```

Comment:
Line 403, need to change `str or {table-like}` to `str, {table-like}`

Reply:
This is updated in commit https://github.com/GenericMappingTools/pygmt/pull/2746/commits/335049ac613dc1e8210bafeba73ccee698178293.